src/Config.php

```diff
@@ -1601,6 +1603,10 @@ public static function getAllConfigData()
         }
 
         include $configFile;
+        if (false === isset($phpCodeSnifferConfig)) {
+            $phpCodeSnifferConfig = null;
```

Comment:
Interesting. Have you come across that situation ?
AFAIK the config file is only supposed to be created & written to by PHPCS, though someone can of course manually edit it. Adding an `unset()` in the file though, seems like a willful act of sabotage... but maybe I'm missing something ?